### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by Luthaf**
Ping @lscalfi, what do you think of this?

### CODE REVIEW

File: src/commands/HBonds.cpp [hunk 1]
```diff
@@ -211,11 +213,11 @@ int HBonds::run(int argc, const char* argv[]) {
...
                     auto theta = angle(r_da, r_dh);
                     if (distance < options.distance && theta < options.angle) {
-                        outfile << frame.topology()[acceptor].name() << " " << acceptor << "   ";
-                        outfile << frame.topology()[donor].name() << " " << donor << "   ";
-                        outfile << frame.topology()[hydrogen].name() << " " << hydrogen << "  : ";
-                        outfile << distance << "    ";
-                        outfile << theta * 180 / pi << std::endl;
+                        outfile << frame.topology()[acceptor].name() << " " << acceptor << "\t";
+                        outfile << frame.topology()[donor].name() << " " << donor << "\t";
+                        outfile << frame.topology()[hydrogen].name() << " " << hydrogen << "\t";
```
Comment (lscalfi) [inline src/commands/HBonds.cpp:217]: is it normal that you erased the `:` here?

**Comment by lscalfi** [comment]
Line 157 of HBonds.cpp, could you add a ` ` after `#`?

File: src/commands/HBonds.cpp (hunk 1 above)
Comment (Luthaf) [reply src/commands/HBonds.cpp:217]: Yes, I prefer using whitespace separators. Do you think it is useful?

Comment (lscalfi) [reply src/commands/HBonds.cpp:217]: Alright, no problem

**Comment by Luthaf** [comment]
The codecov failure is legit: there is nothing to compare this branch against as the master coverage was broken and fixed in this PR.

**Pull request merged**